homeassistant/components/zwave_js/sensor.py

```diff
@@ -39,14 +53,124 @@
+    ENTITY_DESC_KEY_BATTERY: ZwaveSensorEntityDescription(
+        ENTITY_DESC_KEY_BATTERY,
+        device_class=DEVICE_CLASS_BATTERY,
+        state_class=STATE_CLASS_MEASUREMENT,
+    ),
+    ENTITY_DESC_KEY_CURRENT: ZwaveSensorEntityDescription(
+        ENTITY_DESC_KEY_CURRENT,
+        device_class=DEVICE_CLASS_CURRENT,
+        state_class=STATE_CLASS_MEASUREMENT,
+    ),
+    ENTITY_DESC_KEY_VOLTAGE: ZwaveSensorEntityDescription(
+        ENTITY_DESC_KEY_VOLTAGE,
+        device_class=DEVICE_CLASS_VOLTAGE,
+        state_class=STATE_CLASS_MEASUREMENT,
+    ),
+    ENTITY_DESC_KEY_ENERGY: ZwaveSensorEntityDescription(
+        ENTITY_DESC_KEY_ENERGY,
+        device_class=DEVICE_CLASS_ENERGY,
+        state_class=STATE_CLASS_MEASUREMENT,
+    ),
+    ENTITY_DESC_KEY_POWER: ZwaveSensorEntityDescription(
+        ENTITY_DESC_KEY_POWER,
+        device_class=DEVICE_CLASS_POWER,
+        state_class=STATE_CLASS_MEASUREMENT,
+    ),
+    ENTITY_DESC_KEY_POWER_FACTOR: ZwaveSensorEntityDescription(
+        ENTITY_DESC_KEY_POWER_FACTOR,
+        device_class=DEVICE_CLASS_POWER_FACTOR,
+        state_class=STATE_CLASS_MEASUREMENT,
+    ),
+    ENTITY_DESC_KEY_CO: ZwaveSensorEntityDescription(
+        ENTITY_DESC_KEY_CO,
+        device_class=DEVICE_CLASS_CO,
+        state_class=STATE_CLASS_MEASUREMENT,
+    ),
+    ENTITY_DESC_KEY_CO2: ZwaveSensorEntityDescription(
+        ENTITY_DESC_KEY_CO2,
+        device_class=DEVICE_CLASS_CO2,
+        state_class=STATE_CLASS_MEASUREMENT,
+    ),
+    ENTITY_DESC_KEY_HUMIDITY: ZwaveSensorEntityDescription(
+        ENTITY_DESC_KEY_HUMIDITY,
+        device_class=DEVICE_CLASS_HUMIDITY,
+        state_class=STATE_CLASS_MEASUREMENT,
+    ),
+    ENTITY_DESC_KEY_ILLUMINANCE: ZwaveSensorEntityDescription(
+        ENTITY_DESC_KEY_ILLUMINANCE,
+        device_class=DEVICE_CLASS_ILLUMINANCE,
+        state_class=STATE_CLASS_MEASUREMENT,
+    ),
+    ENTITY_DESC_KEY_PRESSURE: ZwaveSensorEntityDescription(
+        ENTITY_DESC_KEY_PRESSURE,
+        device_class=DEVICE_CLASS_PRESSURE,
+        state_class=STATE_CLASS_MEASUREMENT,
+    ),
+    ENTITY_DESC_KEY_SIGNAL_STRENGTH: ZwaveSensorEntityDescription(
+        ENTITY_DESC_KEY_SIGNAL_STRENGTH,
+        device_class=DEVICE_CLASS_SIGNAL_STRENGTH,
+        state_class=STATE_CLASS_MEASUREMENT,
+    ),
+    ENTITY_DESC_KEY_TEMPERATURE: ZwaveSensorEntityDescription(
+        ENTITY_DESC_KEY_TEMPERATURE,
+        device_class=DEVICE_CLASS_TEMPERATURE,
+        state_class=STATE_CLASS_MEASUREMENT,
+    ),
+    ENTITY_DESC_KEY_TIMESTAMP: ZwaveSensorEntityDescription(
+        ENTITY_DESC_KEY_TIMESTAMP,
+        device_class=DEVICE_CLASS_TIMESTAMP,
+        state_class=STATE_CLASS_MEASUREMENT,
+    ),
+    ENTITY_DESC_KEY_TARGET_TEMPERATURE: ZwaveSensorEntityDescription(
+        ENTITY_DESC_KEY_TARGET_TEMPERATURE,
+        device_class=DEVICE_CLASS_TEMPERATURE,
+        state_class=None,
+    ),
+    None: ZwaveSensorEntityDescription("base_sensor"),
```

Comment:
Maybe default to this item in the look up `dict.get` call instead? It makes the types here more consistent.